packages/formation/sass/_shame.scss

```diff
@@ -3,77 +3,58 @@
 // is for this file to be empty, but some times you just have to throw that
 // hack in there to get it working.
 
-// message while loading react app
-.loading-message {
-  padding: 2em 0;
-  text-align: center;
-}
-
 body {
+  // used a lot in vets-website
   .row {
     max-width: 62.5em;
   }
 
   // We have no container element, so elements that are full width have to redefine the container layout every time
   // if they want to line up.
+  // unused in content-library
```

Comment:
Should all these references to content-library say content-build?

Reply:
Great catch!